### ISSUE TITLE
SQL Runner: format SQL on run

### ISSUE THREAD

**Issue by magnew** [lightdash/lightdash#11062]
### Description

Format sql input on run. Maybe there would be an option here.

**Comment by jpetey75** [comment]
yes please make it optional, also https://sqlfmt.com/ might be useful

**Comment by jpetey75** [comment]
The SQL Runner now has a Format SQL button in the editor toolbar that formats your query for the connected warehouse dialect, added in [#21246](https://github.com/lightdash/lightdash/pull/21246). It runs on click rather than automatically on every run, so if that doesn't cover your case, let us know and we'll reopen.
